### PR TITLE
feat (strip_newlines filter): Add strip_newlines filter

### PR DESCRIPTION
**Pull request by jafin**
Add strip_newlines filter  (https://shopify.github.io/liquid/filters/strip_newlines/)    #153

### CODE REVIEW

File: Fluid.Tests/TemplateTests.cs [hunk 1]
```diff
@@ -288,6 +288,8 @@ public Task ShouldEvaluateAssignStatement(string source, string expected)
             return CheckAsync(source, expected);
         }
 
+
```
Comment (hishamco) [inline Fluid.Tests/TemplateTests.cs]: Remove extra line

File: Fluid/Filters/StringFilters.cs [hunk 2]
```diff
@@ -147,6 +149,11 @@ public static FluidValue Strip(FluidValue input, FilterArguments arguments, Temp
             return new StringValue(input.ToStringValue().Trim());
         }
 
+        public static FluidValue StripNewLines(FluidValue input, FilterArguments arguments, TemplateContext context)
+        {
+            return new StringValue(Regex.Replace(input.ToStringValue(),@"\r\n?|\n",""));
```
Comment (hishamco) [inline Fluid/Filters/StringFilters.cs]: Can we avoid using Regex?

File: Fluid.Tests/StringFiltersTests.cs [hunk 3]
```diff
@@ -84,7 +84,23 @@ public void Strip()
...
+        public void StripNewLines()
+        {
+            var input = new StringValue(@"
+Hello
+world
+");
+
+            var arguments = new FilterArguments();
+            var context = new TemplateContext();
+
```
Comment (hishamco) [inline Fluid.Tests/StringFiltersTests.cs]: Remove extra lines

File: Fluid/Filters/StringFilters.cs (hunk 2 above)
Comment (sebastienros) [inline Fluid/Filters/StringFilters.cs]:
```suggestion
            var result = input.ToStringValue();
            
            if (result.Contains("\r"))
            {
                result = result.Replace("\r", "");
            }

            if (result.Contains("\n"))
            {
                result = result.Replace("\n", "");
            }

            return new StringValue(result);
```

Comment (jafin) [reply Fluid/Filters/StringFilters.cs]: Removed Regex replaced with above.

Comment (jafin) [reply Fluid/Filters/StringFilters.cs]: done replaced with search/replace.

**Comment by jptissot** [comment]
fixes #153

**Pull request merged**